src/components/table/SortableTable.tsx

```diff
@@ -114,130 +68,83 @@ function SortableTable(props: SortableTableProps) {
         setPage(0);
     };
 
-    const isSelected = (rowLabel: string) => selected.includes(rowLabel);
-
     // Avoid a layout jump when reaching the last page with empty rows.
     const emptyRows =
         page > 0 ? Math.max(0, (1 + page) * rowsPerPage - rows.length) : 0;
 
     return (
         <Box>
-            <Stack
-                direction="row"
-                spacing={2}
-                sx={{
-                    float: 'right',
-                }}
-            >
-                <ToggleButtonGroup
-                    aria-label="scope of search"
-                    exclusive
-                    size="small"
-                    value={searchScope}
-                    onChange={handleAlignment}
-                >
-                    <ToggleButton aria-label="search all" value="all">
-                        All
-                    </ToggleButton>
-                    <ToggleButton aria-label="search active" value="active">
-                        Active
-                    </ToggleButton>
-                    <ToggleButton aria-label="search all paused" value="paused">
-                        Paused
-                    </ToggleButton>
-                </ToggleButtonGroup>
-                <TextField
-                    id="outlined-search"
-                    label="Search for keyword"
-                    type="search"
-                    variant="filled"
-                />
-            </Stack>
```

Comment:
Removing the fake search / filtering stuff as we don't have any true use for it yet.